pom.xml

```diff
@@ -6599,6 +6601,10 @@
 
     <profile>
       <id>sonar</id>
+      <properties>
+        <failIfNoTests>false</failIfNoTests>
```

Comment:
It was indeed a property set by the job.
It's a global safety to avoid try and errors, which will end up adding the property on specific jobs that will probably be left even when adding tests.
I'd rather leaving the property at this level.